src/qibolab/transpilers/general_connectivity.py

```diff
@@ -190,7 +190,7 @@ def translate_circuit(self, qibo_circuit):
                 gate_qubits.sort()
                 gate_qubits.append(index)
                 translated_circuit.append(gate_qubits)
-            if len(gate.qubits) >= 3:
+            if len(gate.qubits) >= 3:  # pragma: no cover
```

Comment:
Also this one is better to cover: you can use `pytest.raises`. For the same reason you want to test regular code, you also want to test errors (to check they are properly raised when intended).